deeprankcore/query.py

```diff
@@ -236,11 +280,12 @@ def process( # pylint: disable=too-many-arguments, too-many-locals
         """
         Args:
             prefix (Optional[str], optional): Prefix for the output files. Defaults to None, which sets ./processed-queries- prefix.
-            feature_modules(Union[ModuleType, List[ModuleType]], optional): Features' module or list of features' modules used to generate features. 
-                Each feature's module must implement the :py:func:`add_features` function, and features' modules can be found (or should be placed
-                in case of a custom made feature) in `deeprankcore.features` folder. 
-                Defaults to None, which means that all available modules in `deeprankcore.features` are used to generate
-                the features. 
+            feature_modules (Optional[Union[ModuleType, List[ModuleType], str, List[str]]], optional): Features' module or list of features' modules
+                used to generate features (given as string or as an imported module). Each module must implement the :py:func:`add_features` function, 
+                and features' modules can be found (or should be placed in case of a custom made feature) in `deeprankcore.features` folder. 
+                Use 'all' to include all feature modules.
+                If set to None or 'all', all available modules in `deeprankcore.features` are used to generate the features.
```

Comment:
I would say that only if set to 'all' all the module will be used

Reply:
Yeah, this was a typo.